artifacts/definitions.py

```diff
@@ -40,7 +40,6 @@
     'IM': 'Instant Messaging / Chat applications artifacts.',
     'iOS': 'Artifacts related to iOS devices connected to the system.',
     'History Files': 'History files artifacts e.g. .bash_history.',
-    'KnowledgeBase': 'Artifacts used in knowledge base generation.',
```

Comment:
This might not be correct anymore.  I removed this one because it was a duplicate key, but it appears that someone may have removed the other one as the duplicate key leaving this one in instead.